static/scss/toast.scss

```diff
@@ -20,6 +20,9 @@
 
     .toast-body {
       padding: 0 10px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
```

Comment:
I don't think you need to set `flex-direction: column;` here, I get the same display if I just do

```scss
display: flex;
align-items: center;
```